packages/compiler/src/render3/view/i18n/get_msg_utils.ts

```diff
@@ -22,7 +22,25 @@ export function createGoogleGetMsgStatements(
   const messageString = serializeI18nMessageForGetMsg(message);
   const args = [o.literal(messageString) as o.Expression];
   if (Object.keys(params).length) {
-    args.push(mapLiteral(params, true));
+    // Message template parameters.
```

Comment:
I'd propose adding a quick example of the generated code that'd be produced (so it's easier to navigate the code). Should we also rename `params` -> `placeholders`?

```suggestion
    // i18 message placeholders object, e.g. `{"interpolation": "...", "startTagSpan": "..."}`
```

Reply:
Renamed to `placeholderValues` and added a doc comment that shows an example Angular template and the `goog.getMsg()` statement it generates.